packages/zowe-explorer-ftp-extension/src/extension.ts

```diff
@@ -10,7 +10,7 @@
  */
 
 import * as vscode from "vscode";
-import { ZoweExplorerApi, ZoweVsCodeExtension } from "@zowe/zowe-explorer-api";
+import { ZoweVsCodeExtension } from "@zowe/zowe-explorer-api";
```

Comment:
@katelynienaber did you do any changes in the FTP extension?

Reply:
I was trying the new code there. I removed all the other changes, but this import is not being used anywhere in the extension.ts file, so I removed it. I can put it back if it makes reviewing easier.